tests/conftest.py

```diff
@@ -2797,6 +2798,20 @@ def fio_job_dict_session():
     return fio_artefacts.get_job_dict()
 
 
+@pytest.fixture(scope="function")
+def start_apps_workload(request):
+    """
+    Application workload fixture which reads the list of app workloads to run and
+    starts running those iterating over the workloads in the list for a specified
+    duration
+
+    Usage:
+    start_app_workload(workloads_list=['pgsql', 'couchbase', 'cosbench'], run_time=60,
+    run_in_bg=True)
+    """
+    return start_app_workload(request)
```

Comment:
Any errors or exceptions if installation of any one/all workload is unsuccessful ??

Reply:
Yes, if any of the workload fails the workload will throw its own defined exception/error